lms/djangoapps/shoppingcart/models.py

```diff
@@ -387,7 +387,7 @@ def send_confirmation_emails(self, orderitems, is_order_type_business, csv_file,
                         'recipient_type': recipient[2],
                         'site_name': site_name,
                         'order_items': orderitems,
-                        'course_names': ", ".join([course_info[0] for course_info in courses_info]),
+                        'course_names': ", ".join([course_info for course_info in courses_info]),
```

Comment:
Once again, no need to make a temporary list copy of `course_info` here.
Can just do `", ".join(courses_info)`.